### ISSUE TITLE
Adding the admin columns to Users list in WP dashboard as we have for CPTs

### ISSUE THREAD

**Issue by xtofr** [CrocoBlock/suggestions#6380]
Hi !
A quick suggestion concerning user accounts with metafields.
Is it possible - as it is possible for CPT - to add admin columns containing metafields values in the standard WP Dashboard /users
By default we have :
Username     Name        Email      Role       Posts

It would bo cool to add a taxonomy for instance, or the renewal date of subscription... whatever metafield added to the user profile.

Thanks

**Comment by azimdotnet** [comment]
Yes, It will be useful to view custom data of user and please add user admin filter also.

![User admin](https://user-images.githubusercontent.com/26623136/219616644-e860d0b6-8953-49fb-b5fb-6778317b7540.png)

For now I use mange user coloum to add admin user column, its work  on meta field also.
https://wordpress.org/plugins/manage-user-columns/
